tdms/include/globals.h

```diff
@@ -63,17 +55,13 @@ enum RCSType   { parallel , perpendicular };
 //			Mathematical Constants
 // **************************************
 
-
 const double dcpi  = 3.14159265358979;

```

Comment:
It wasn't touched in this PR, but
```suggestion
const double dcpi  = 3.141592653589793;
```
is a more accurate double-precision representation of pi (any more digit would be redundant).  Alternatively, include `math.h` and use `M_PI` without reinventing the wheel.

Reply:
sounds good 👍🏼

Reply:
or perhaps not: [build now fails on Windows](https://github.com/UCL/TDMS/runs/7968328592?check_suite_focus=true#step:10:40 )